Python/PDF_Reader/pdf_reader.py

```diff
@@ -0,0 +1,42 @@
+# importing required libraries
+import pyttsx3
+import PyPDF2
+
+# initializing the speaker
+speaker = pyttsx3.init()
+
+# opening the document in read mode. You can mention your document name here
+book = open("sample_pdf.pdf", 'rb')
```

Comment:
For the file path of the book, make it so that user can provide the input for the file.

Reply:
Could you elaborate on this change? I have understood the remaining ones but what path should I provide for the book?

Reply:
Here you have provided `sample_pdf.pdf`. But the user running the script should be able to provide any file that they want.
So, for the book, take user input of the file path for the book.